sqs-tester/src/main/java/uk/gov/justice/laa/crime/microservice/sqstester/HelloController.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.justice.laa.crime.microservice.sqstester;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HelloController {
```

Comment:
I would try adding a single integration test for the controller. Once done you will be able to quickly test if any change(s) you are making are causing any breakages without having to manually start the service etc.

See this class `CrimeApplicationIntegrationTest` as an example (your test will be significantly simpler than this!)

https://github.com/ministryofjustice/laa-crime-applications-adaptor/blob/8c820f38f4acce8418bef3af05d665c951869309/crime-applications-adaptor/src/test/java/uk/gov/justice/laa/crime/applications/adaptor/integration/CrimeApplicationIntegrationTest.java

Reply:
Changes look good